### ISSUE TITLE
discuss Langevin

### ISSUE THREAD

**Issue by paciorek** [nimble-dev/nimbleHMC#5]
I had done some work a while back to implement a version of Langevin (in nimble branch `extend_langevin`) that adapts the full covariance following a discussion with Giacomo Zanella (https://arxiv.org/abs/1908.11812). I hadn't gotten acceptable performance with that or (I believe) with the basic Langevin on my test cases and hadn't gotten a chance to get back to it to explore further. 

I'm wary of exposing the current Langevin in `nimbleHMC` until we do more exploration of performance/behavior.  One thought is that we remove it entirely for now, while another is that we don't export it.

@danielturek @perrydv, thoughts?

**Comment by danielturek** [comment]
@paciorek  I'd be fine with the non-export option.

Also noting that the `nimbleHMC` repo `README` says: 
- Langevin sampler (under development)

**Comment by paciorek** [comment]
Great, I'll leave it to you to do that.

**Comment by danielturek** [comment]
`sampler_langevin` is no longer exported in the `nimbleHMC` package.

I'm leaving this issue open, for discussion about re-working / improving the langevin sampler.

**Comment by danielturek** [comment]
@paciorek Yes, that sounds reasonable to me.  I'm happy to make this change (remove Roxygen for `sampler_langevin`) if we're in agreement here.

**Comment by paciorek** [comment]
'removal' could also simply be "commenting out" the roxygen "comments" by adding another level of comment hashes, so as to keep the info in the code file.

**Comment by danielturek** [comment]
@paciorek Yes absolutely, that was my intention.  I'll do this right now.  Keep this issue open, though.
